.github/workflows/conventional-commit-pr-title.yml

```diff
@@ -0,0 +1,17 @@
+name: Validate PR Title
```

Comment:
Will you also enforce "Squash and Merge" in the repo settings?

Reply:
Done